ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/HighAvailabilityTestUtil.java

```diff
@@ -264,5 +270,121 @@ public static void sendLagReportingRequest(
           .get();
     }
   }
+
+  static String extractQueryId(final String outputString) {
+    final java.util.regex.Matcher matcher = QUERY_ID_PATTERN.matcher(outputString);
+    if (!matcher.find()) {
+      throw new AssertionError("Could not find query id in: " + outputString);
+    }
+    return matcher.group(1);
+  }
+
+  // Ensures that lags exist for the cluster.  Makes the simplified assumption that there's just one
+  // state store.
+  static BiFunction<KsqlHostInfoEntity, Map<KsqlHostInfoEntity, HostStatusEntity>, Boolean>
```

Comment:
This method checks that the clusterStatus response contains lag reporting? Not that there is actual lag, right?

Reply:
Correct.  The name and comments weren't great.  Changed to `lagsReported`. The latter version checks the actual values.